database_schema.md

```diff
@@ -136,6 +136,76 @@ seat_id: {
 }
 ```
 
+### Minutes Item
+```
+minutes_item: {
+    minutes_item_file_id: str
+    minutes_item_id: str
+    name: str
+    uri: str
+    title: str
+    matter: {
+      matter_id: str
+      matter_name: str
+      matter_title: str
+    }
+    created: datetime
+}
+```
+
+### Event Minutes Item
+```
+event_minutes_item: {
+    event_minutes_item_id: str
+    event_id: str
+    minutes_item_id: str
+    index: int
+    decision: str
+    matter: {
+      matter_id: str
+      matter_name: str
+    }
+    votes: [
+        {
+            vote_id: str
+            person_id: str
+            person_name: str
+            vote_decision: str
+        }
+    ]
```

Comment:
Can be entirely optional if there was not vote on the item.

Reply:
Should I put in anything to mark the optional stuff?

Reply:
Uhhhh probably but I think there are a lot of places that we missed this in. So I will try to go back and mark stuff as optional where I see it in a later PR.